src/EnergyPlus/WaterThermalTanks.cc

```diff
@@ -811,6 +811,46 @@ namespace WaterThermalTanks {
 	{
 
 		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Richard Raustad, FSEC
+		//       DATE WRITTEN   July 2016
+		//       MODIFIED       na
```

Comment:
For new functions, omit unused comments like this (all the na's and an headings that have na under them), and keep the other comment minimal.